README.md

```diff
@@ -144,6 +144,21 @@ http://dockerhost:5000
 When you install docker-osx-dev, it adds an entry to your `/etc/hosts` file so
 that `http://dockerhost` works as a URL for testing your Docker containers.
 
+# docker-machine support
+
+docker-machine support is experimental. You can use it as they way it is used for
+Boot2Docker, but provide `DOCKER_MACHINE_NAME=<machine-name>` at the begging of
+the `docker-osx-dev`. So as an example, run as:
+
+```
+> cd /foo/bar
+> DOCKER_MACHINE_NAME=foo docker-osx-dev
```

Comment:
Any reason to use an environment variable here instead of a flag? It seems more consistent with the docker-osx-dev design to do something like:

```
docker-osx-dev --machine-name foo
```